core-bundle/src/Resources/contao/pages/PageRegular.php

```diff
@@ -221,7 +221,7 @@ protected function prepare($objPage)
 
 		// Assign the title and description
 		$this->Template->title = strip_tags($this->replaceInsertTags($objLayout->titleTag));
```

Comment:
I don’t think so.

Reply:
We almost exclusively use the new methods to convert something from user input for one of the meta attributes (title, description). In this case, that logic wouldn't be applied if I use anything but the standard `{{page::pageTitle}}` insert tag?

Reply:
`$this->Template->title` needs to be encoded, the new methods are used to get the “decoded” raw value.

Reply:
Then maybe I don't understand the purpose of the new methods at all. Isn't that exactly what https://github.com/contao/contao/pull/3014/files#diff-0d3a685427658e56073dd7043ebcaa18132e0c77bc6cd9bb56409274f2503f6dR207 does a few lines above? The user input is first converted to plain text through the new methods, and then encoded with `htmlspecialchars`?

Reply:
The new methods are meant to be used everywhere where we want “raw” non-encoded data. In new data objects like the response context, we want to already use raw data instead of input-encoded data. In the far future we want “raw” non-encoded data everywhere and only encode as a final step in the template rendering code.